openapi/components/examples/lambdas/200-wearables-paginated.json

```diff
@@ -0,0 +1,93 @@
+{
+    "value": {
+        "wearables": [
+            {
+                "urn": "urn:decentraland:matic:collections-v2:0x6d83706836e820e79252edd48da419d2d03592d4:0",
+                "individualData": [
```

Comment:
Are we missing this property in the schema?

Reply:
yes thanks!